### PR TITLE
Raider scs

### PR DESCRIPTION
**Pull request by tksolway**
Hey Chris, Here are all the raiders with proper hit charts.  I also took the most up to date Autoload (from 5pm today) and added all the new raiders too it.  Let me know if there's any changes you think need to be made.

### CODE REVIEW

**Comment by AncientSion** [comment]
So basicly, all of those vessels now have either a new hitArray or the "old" one is correct with the "shorter" array-indexing and/or name to displayName conversion ?

**Comment by AncientSion** [comment]
Haha :P you made some errors. Cant blame you tho, there was so much talk about it back then, really hard to get the current state properly.
I will merge the PR and fix it up later today.

Whats wrong:

We want to use displayName in the hitArray.

$displayName really is the actual name for displaying the weapon in the GUI.
I.e. "standard particle beam" is almost right, but it needs proper upper-case -> "Standard Particle Beam", same for basicly anything "reactor" -> "Reactor" and "light laser" -> "Light Laser";

Im going to quickly explain why "i" made it like this because it might be more easy to understand and remember it for future causes.

hitTable is an array that is used on the client to decide which system to hit. The game then loops over a ships sstems and compares the entry it rolled in the hitTable array to the system object (which has $name, $displayName, $id etc)
So, why settle with displayName for the comparision:
'''
if ($system->displayName  == $hitArray[roll]){
    return $system // and damage it)
}'''

I use the hitTable array to populate the "Hit Chart" table overview in the shipwindow AS WELL. 
By filling the hitTable with the actual DISPLAY NAME, i.e. proper uppercase etc, i can iterate over the hitTable and use it to fill the GUI.
If we would use "name", the table in the GUI wouldnt look as nice by default.
 Instead, i would have to iterate over the hitTable array and convert the "names" into the actual weapons displayName so the table in the GUI would make "sense" to a human player.

Long story short, by using displayName i save a lot whole of conversion loops on the client.

I know its a mess - still does this make sense to you ?

**Comment by tksolway** [comment]
Makes sense, I was under the impression we would be normalizing the displayName (changing the text to all uppercase, or lower case) in order to remove the need to match the case exactly.  Otherwise the names should be correct.  For the ones I had finished before we'd decided on using the shortened hit chart, I just left them long, since it shouldn't affect the algorithm.

PHP has a built in function strtoupper that returns the given string as all uppercase.  so you should be able to just do this and not alter anything in all the hit charts:

if (strtoupper($system->displayName)  == strtoupper($hitArray[roll]))

**Comment by AncientSion** [comment]
Please do continue with short hitArrays and proper capitalization. I dont want to use build functions unless its absolutly necesarry. It makes me think the code, or rather input, is flawed.

Note that there are weapons who dont fit the theme.
i.e. pulse and plasma are called "Cannon" @ the end.
Lasers, however, are just "Laser" @ the end.

I will look over your stuff, so a second pair of eyes is involved.
Also, i will eventually either make Lasers being "Cannons" or get rid of all "Cannon" substrings.

I absolutly require coherence in code :D :package.

**Comment by tksolway** [comment]
No Problem, Let me know if you make any changes to the Cannons or Cannon strings, otherwise I'll start on the Civilian SCSs now.
